index.html

```diff
@@ -185,7 +185,7 @@
         <h3>Research interests</h3>
         <p class="research_interests">Learning Theory; Optimization; Deep Learning; Adversarial Models; </p>
         <h3>About me</h3>
-        <p>I have an undergraduate degree from IIT Kanpur with a B.Tech in Computer Science and Engineering and a minor in Linguistics Theory. During these four years, I visited quite a few places where I carried out short term research projects. I spent some time in Indian Statistical Institute, Kolkata and Indian Institute of Science, Bengaluru. Following that I spent a summer at the Amazon ML lab in Bangalore where I worked on developing a model in competitive pricing, which was later deployed(supposedly). I spent the next summer in the Montreal Institute of Learning Algorithms(MILA) where I worked on developing an adaptive segmentation language generation model, which was published at the MLSLP workshop at ICML, Sydney. I spent the following summer at the Twitter Cortex group in NYC working with Nicholas Leonard and Nicolas Koumchatzky woking on adversarial discrete sequence generation models.
+	      <p> Here is my <a href="https://amartya18x,github.io">personal webpage!</a>. I have an undergraduate degree from IIT Kanpur with a B.Tech in Computer Science and Engineering and a minor in Linguistics Theory. During these four years, I visited quite a few places where I carried out short term research projects. I spent some time in Indian Statistical Institute, Kolkata and Indian Institute of Science, Bengaluru. Following that I spent a summer at the Amazon ML lab in Bangalore where I worked on developing a model in competitive pricing, which was later deployed(supposedly). I spent the next summer in the Montreal Institute of Learning Algorithms(MILA) where I worked on developing an adaptive segmentation language generation model, which was published at the MLSLP workshop at ICML, Sydney. I spent the following summer at the Twitter Cortex group in NYC working with Nicholas Leonard and Nicolas Koumchatzky woking on adversarial discrete sequence generation models.
```

Comment:
I tested this locally and your personal website link didn't work. Your link should be `https://amartya18x.github.io/` (i.e. full-stop instead of comma before github).